app/Agency.php

```diff
@@ -133,6 +133,25 @@ static function whereHasApplicant($data, $request)
         });
     }
 
+    /**
+     * Search Report Scope function
+     *
+     * Data search can be done by entering "search" in the form of the agency ID or Applicant Name
+     *
+     * @param $query
+     * @param $request
+     * @return $query
+     */
+    public function scopeSearchReport($query, $request)
+    {
+        return $query->when($request->has('search'), function ($query) use ($request) {
```

Comment:
```suggestion
        return $query->when($request->has('search'), function ($query) use ($request) {
                          $query->where(function ($query) {
                             $query->where('agency.id', 'LIKE', "%{$request->input('search')}%")
                                     ->orWhereHas('applicant', function ($query) use ($request) {
                                          $query->where('applicant_name', 'LIKE', "%{$request->input('search')}%");
                                 })
                  )};
        });
```

untuk search ini mungkin di tambahkan where parent nya kang @rindibudiaramdhan untuk ngebungkus $query->where di bawah nya

Reply:
![image](https://user-images.githubusercontent.com/41193120/112807320-f2a58400-90a1-11eb-82e8-ee23cef7244a.png)
contoh nya gini kang @rindibudiaramdhan

Reply:
sip sudah ditambahkan `where` parentnya